packages/components/src/radio-control/README.md

```diff
@@ -1,11 +1,59 @@
 # RadioControl
 
-RadioControl component is used to generate radio input fields.
+Use radio buttons when you want users to select one option from a set, and you want to show them all the available options at once.
 
+![](https://make.wordpress.org/design/files/2018/11/radio.png)
+Selected and unselected radio buttons
 
-## Usage
+## Table of contents
+
+1. [Design guidelines](#design-guidelines)
+2. [Development guidelines](#development-guidelines)
+3. [Related components](#related-components)
+
+## Design guidelines
+
+### Usage
+
+#### When to use radio buttons
+
+Use radio buttons when you want users to:
+
+- Select a single option from a list.
+- Expose all available options.
+
+If you have a list of available options that can be collapsed, consider using a dropdown menu instead, as dropdowns use less space. A country selection field, for instance, would be very large as a group of radio buttons and wouldn't help the user gain more context by seeing all options at once.
+
+![](https://make.wordpress.org/design/files/2018/11/radio-usage-do.png)
+**Do**
+Use radio buttons when only one item can be selected from a list.
+
+![](https://make.wordpress.org/design/files/2018/11/radio-usage-dont.png)
+**Don’t**
+Don’t use checkboxes when only one item can be selected from a list. Use radio buttons instead.
+
+#### Defaults
+
+When using radio buttons **one should be selected by default** (i.e., when the page loads, in the case of a web application).
+
+**User control**
```

Comment:
These are headers, not paragraph text; they should be labelled with header markdown (eg `##### User control`), not bold text like this. 😄